src/app.js

```diff
@@ -81,12 +82,13 @@ app.get('/oauth/authorize', (req, res) => {
   res.redirect(auth_url);
 });
 
-app.get('/oauth/redirect', async (req, res) => {
+app.get('/oauth/redirect', async(req, res) => {
   const { code, state } = req.query;
   try {
     await getSecretsAndSave(code);
     res.send('Thank you, you have successfully authenticated your team!');
-  } catch (err) {
+  }
```

Comment:
why changing the indentation here?

Reply:
c9 automatically changes all the indents to this when I save a file